build/release/containers.sh

```diff
@@ -15,7 +15,7 @@ build() {
     trap "rm -fr $tmpdir" EXIT
     cat <<EOF > $tmpdir/Dockerfile
 FROM alpine:3.5
-RUN apk add --no-cache gptfdisk util-linux coreutils
+RUN apk add --no-cache gptfdisk util-linux coreutils e2fsprogs
```

Comment:
Our Alpine image doesn't have `mkfs` in it, this package adds support for ext filesystems (e.g., ext4) via `mkfs`.

Without this package, `mkfs` fails:
```
mkfs /dev/disk/by-partuuid/117d4e9a-811b-4363-8f73-89db2c99fcc0
mkfs: failed to execute mkfs.ext2: No such file or directory
```